docs/es/docs/deployment/docker.md

```diff
@@ -118,62 +118,62 @@ $ docker run -d --name mycontainer -p 80:80 myimage
+Deberías de poder revisarlo en el URL de tu contenedor de Docker, por ejemplo: <a href="http://192.168.99.100/items/5?q=somequery" class="external-link" target="_blank">http://192.168.99.100/items/5?q=somequery</a> or <a href="http://127.0.0.1/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1/items/5?q=somequery</a> (o equivalente, usando tu Docker host)
 
-You will see something like:
+Verás algo como esto:
 
 ```JSON
 {"item_id": 5, "q": "somequery"}
 ```
 
-## Interactive API docs
+## Documentación Interactiva de la API
 
-Now you can go to <a href="http://192.168.99.100/docs" class="external-link" target="_blank">http://192.168.99.100/docs</a> or <a href="http://127.0.0.1/docs" class="external-link" target="_blank">http://127.0.0.1/docs</a> (or equivalent, using your Docker host).
+Ahora puedes ir a <a href="http://192.168.99.100/docs" class="external-link" target="_blank">http://192.168.99.100/docs</a> o <a href="http://127.0.0.1/docs" class="external-link" target="_blank">http://127.0.0.1/docs</a> (o equivalente, usando tu Docker host).
 
-You will see the automatic interactive API documentation (provided by <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>):
+Veras la documentación interactiva automática de tu API (proveída por <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>):
 
 ![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
 
-## Alternative API docs
+## Documentación Alternativa de la API
 
-And you can also go to <a href="http://192.168.99.100/redoc" class="external-link" target="_blank">http://192.168.99.100/redoc</a> or <a href="http://127.0.0.1/redoc" class="external-link" target="_blank">http://127.0.0.1/redoc</a> (or equivalent, using your Docker host).
+Ademas puedes ir a <a href="http://192.168.99.100/redoc" class="external-link" target="_blank">http://192.168.99.100/redoc</a> o <a href="http://127.0.0.1/redoc" class="external-link" target="_blank">http://127.0.0.1/redoc</a> (o equivalente, usando tu Docker host).
 
-You will see the alternative automatic documentation (provided by <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
+Veras la documentación alternativa automática de tu API (proveída por <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
 ## Traefik
 
-<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> is a high performance reverse proxy / load balancer. It can do the "TLS Termination Proxy" job (apart from other features).
+<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> es un balanceador de carga/proxy inverso de alto rendimiento. Puede hacer la tarea de "TLS Termination Proxy" (ademas de otras características).
```

Comment:
`job` means `trabajo`

Reply:
I think that in this case tarea still work, do you think that: 
```suggestion
<a href="https://traefik.io/" class="external-link" target="_blank">Traefik</a> es un balanceador de carga/proxy inverso de alto rendimiento. Puede hacer el trabajo de "TLS Termination Proxy" (ademas de otras características).
```
works better?, 🤔

Reply:
Yes, 'tarea' sounds very well!